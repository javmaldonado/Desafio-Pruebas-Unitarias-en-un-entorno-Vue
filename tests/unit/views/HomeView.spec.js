import { mount } from "@vue/test-utils";
import HomeView from "@/views/HomeView.vue";


describe("Tests de vista Contact", () => {
    it("Test1 - Snapshot de la vista Contact  se crea correctamente", () => {
    
      const wrapper = mount(HomeView);  // Seleccionar el componente que será puesto a prueba //
      expect(wrapper.html()).toMatchSnapshot();  // Realiza una afirmación sobre la prueba si  hay Match pasa la prueba //
    });
  
  });
  