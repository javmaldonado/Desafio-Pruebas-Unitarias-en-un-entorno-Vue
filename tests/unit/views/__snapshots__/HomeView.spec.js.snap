// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests de vista Contact Test1 - Snapshot de la vista Contact  se crea correctamente 1`] = `
<div class="home">
  <h1>This is an Home page</h1>
</div>
`;
